### PR TITLE
Watchdog fix

### PR DESCRIPTION
**Pull request by eurich**

### CODE REVIEW

File: router-watchdog.js [hunk 1]
```diff
@@ -58,11 +58,10 @@ function pingEndpoints() {
       if (failCounter >= CONFIG.numberOfFails) {
         print("Too many fails, resetting...");
         failCounter = 0;
-        Timer.clear(pingTimer);
```
Comment (Mircho) [inline router-watchdog.js:61]: This should stay. As the code listens for the notification of the switch turning back to on and then turns the timer back

File: router-watchdog.js [hunk 2]
```diff
@@ -58,11 +58,10 @@ function pingEndpoints() {
...
         //set the output with toggling back
         Shelly.call(
           "Switch.Set",
-          { id: 0, on: true, toggle_after: CONFIG.toggleTime },
+          { id: 0, on: false, toggle_after: CONFIG.toggleTime },
```
Comment (Mircho) [inline router-watchdog.js:64]: Should also be changed in the notification handler at the end of the file where the the state of the switch is monitored.

**Review by Mircho — CHANGES_REQUESTED** [review]
Thanks for the proposal. Just some minor comments

File: router-watchdog.js (hunk 2 above)
Comment (eurich) [reply router-watchdog.js:65]: done

File: router-watchdog.js (hunk 1 above)
Comment (eurich) [reply router-watchdog.js]: re-added the code as requested.

**Review by Mircho — APPROVED** [review]

**Pull request merged**